relay-server/src/envelope/item.rs

```diff
@@ -621,6 +627,7 @@ impl ItemType {
             Self::ReplayVideo => "replay_video",
             Self::CheckIn => "check_in",
             Self::Log => "log",
+            Self::TraceMetric => "metric",
```

Comment:
```suggestion
            Self::TraceMetric => "trace_metric",
```

Like we talked about for data categories etc. I think we should commit to "trace metric", less confusion and possibilities to mix it with DDM or metrics how they still exist in Relay for dynamic sampling.